.about.yml

```diff
@@ -20,8 +20,8 @@ licenses:
     url: https://github.com/18F/about_yml/blob/master/LICENSE.md
 tasks: https://github.com/18F/ffd-info-exchange/issues#boards
 contact:
-- url: mailto:ffd-info-exchange@gsa.gov #replace once we have a google group
-  text: Federal Front Door Information Exchange #replace once we have a google group
+- url: mailto:ffd-info-exchange@gsa.gov
+  text: Federal Front Door Information Exchange
 team:
-- github: 18F/ffd-info-x
+- github: @18F/ffd-info-x
```

Comment:
Two questions:
* **Semantics:** Is this '@' drawn from documentation/another `.about.yml` file? If not, the direct link for the team is https://github.com/orgs/18F/teams/ffd-info-x.
* **Future-proofing:** How archival are these GitHub teams? I imagine 18Fers who leave are removed from any 18F-related GitHub teams. If that's the case, it may be clearer and more accurate to list names/GitHub usernames/etc. instead.

I haven't linked to a GitHub team before in an `.about.yml` file, so feel free to point me to further context.